apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -1164,7 +1175,15 @@ export class InstallManager {
         args.push('--verbose');
       }
     } else if (this._rushConfiguration.packageManager === 'pnpm') {
-      args.push('--store', this._rushConfiguration.pnpmStoreFolder);
+      // Only explicitly define the store path if `pnpmStore` is using the default, or has been set to
+      // 'local'.  If `pnpmStore` = 'global', then allow PNPM to use the system's default
+      // path.  In all cases, this will be overridden by RUSH_PNPM_STORE_PATH
+      if (
+        this._rushConfiguration.pnpmOptions.pnpmStore === 'local' ||
+        EnvironmentConfiguration.pnpmStorePathOverride
+      )
+          args.push('--store', this._rushConfiguration.pnpmOptions.pnpmStorePath);
+      }
```

Comment:
```suggestion
      ) {
        args.push('--store', this._rushConfiguration.pnpmOptions.pnpmStorePath);
      }
```

Reply:
Thank you for catching that!  I don't know how I missed it